security/authorization/compiler/README.md

```diff
@@ -0,0 +1,65 @@
+# User Policy Compiler
+
+The User Defined Policy is written in a specific yaml format. An example Yaml File is listed below.
+
+action: ALLOW
+rules:
+- name: test access
+
+  condition: request.url_path.startsWith('/pkg.service/test')
```

Comment:
So for a single Name there can be multiple condition: lines or do you mean condition: c1 && c2?

Reply:
Since if there can be multiple condition lines in the yaml I have to go in and change the resulting input struct and how everything gets converted to the cel.Expr